staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -334,8 +339,9 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 			if err != errorStopRequested {
 				switch {
 				case isExpiredError(err):
-					r.setIsLastSyncResourceVersionExpired(true)
-					klog.V(4).Infof("%s: watch of %v ended with: %v", r.name, r.expectedTypeName, err)
+					// FIXME: the fact I can't watch doesn't mean I can't list from that point
```

Comment:
is that accurate?

Reply:
well - the comment isn't
what I wanted to say is that assuming the official semantic list with RV=<rv> meaning "result is at least as fresh as <rv>", we can list from visibly further in the future in many cases (and we can't start the watch from exactly that point).